src/base.py

```diff
@@ -128,10 +126,10 @@ def from_layoutparser(
         text_block = text_block.to_rectangle()
 
         new_format_coordinates = [
-            (text_block.block.x_1, text_block.block.y_1),
-            (text_block.block.x_2, text_block.block.y_1),
-            (text_block.block.x_2, text_block.block.y_2),
-            (text_block.block.x_1, text_block.block.y_2),
+            (text_block.block.x_1, text_block.block.y_1),  # type: ignore
```

Comment:
Why the type ignore here?

Reply:
Don't know. Kalyan added need to check with him. Leaving for now.

Reply:
The comments around the code block explain it: 

https://github.com/climatepolicyradar/navigator-document-parser/blob/a4d392ec5a4304c612c7e9162ae4c5f98246b7c2/src/base.py#L163

That sounds like a problem and something we shouldn't ignore? 
@kdutia @stefl14 

          # Convert from a potentially not rectangular quadrilateral to a rectangle.
                  # This method does nothing if the text block is already a rectangle.
                  text_block = text_block.to_rectangle()
          
                  new_format_coordinates = [
                      (text_block.block.x_1, text_block.block.y_1),  # type: ignore
                      (text_block.block.x_2, text_block.block.y_1),  # type: ignore
                      (text_block.block.x_2, text_block.block.y_2),  # type: ignore
                      (text_block.block.x_1, text_block.block.y_2),  # type: ignore
                  ]
          
                  # Ignoring types below as this method will raise an error if any of these values are None above.
                  return PDFTextBlock(
                      text=[text_block.text],  # type: ignore
                      text_block_id=text_block_id,  # e.g. p0_b3
                      coords=new_format_coordinates,  # type: ignore
                      type_confidence=text_block.score,  # type: ignore
                      type=text_block.type,  # type: ignore
                      page_number=page_number,
                  )

Reply:
@THOR300 I think that comment refers to the `type: ignore` comments below it.

When i uncomment the type: ignore I get the following error in my IDE.
```
Cannot access member "x_1" for type "BaseCoordElement"
  Member "x_1" is unknownPylancereportGeneralTypeIssues
```

It's to do with a typing issue in layoutparser: the typechecker recognises `text_block.block` as type `BaseCoordElement` but it's actually `Rectangle`, which has the properties `x_1` etc.

So, I think these ignores are fine - you might want to clarify why with a comment on the code though. 

As to the type ignores you were referring to mark - i'm not sure why they're there

Reply:
@kdutia I made a quick PR here that looks like it's passed without failures so assuming it's just an IDE hint? 
My IDE doesn't throw a hint - pycharm CE 2022.2.

Reply:
https://github.com/climatepolicyradar/navigator-document-parser/pull/64